quiz.css

```diff
@@ -0,0 +1,115 @@
+
+.article_1 {
+  width: 30%;
+  border: 1px solid lightblue;
+  margin: 2.5%;
+  float: left;
+}
+
+.article_2 {
```

Comment:
Can you combine all your article styles into a single styles? Also, read a little more about how floats work. Hint: you should `float: left` all articles